pimd/pim_cmd.c

```diff
@@ -8251,12 +8252,12 @@ DEFUN (interface_ip_igmp_query_max_response_time,
 
 DEFUN (interface_no_ip_igmp_query_max_response_time,
        interface_no_ip_igmp_query_max_response_time_cmd,
-       "no ip igmp query-max-response-time (10-250)",
+       "no ip igmp query-max-response-time [(1-65535)]",
        NO_STR
        IP_STR
        IFACE_IGMP_STR
        IFACE_IGMP_QUERY_MAX_RESPONSE_TIME_STR
-       "Time for response in deci-seconds\n")
+       IGNORED_IN_NO_STR)
 {
 	char default_query_max_response_time[4];
 
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
chnaged to destroy